store/tikv/2pc.go

```diff
@@ -598,14 +599,18 @@ func (tm *ttlManager) close() {
 }
 
 func (tm *ttlManager) keepAlive(c *twoPhaseCommitter) {
-	// Ticker is set to 1/3 of the PessimisticLockTTL.
-	ticker := time.NewTicker(time.Duration(PessimisticLockTTL) * time.Millisecond / 3)
+	const lockTTL = 3000 // 3s
```

Comment:
This is too short.

Reply:
Now the value is updated to PessimisticLockTTL